### ISSUE TITLE
Notify user on door access failure

### ISSUE THREAD

**Issue by tswfi** [TampereHacklab/mulysa#230]
If door access fails, send email to user of the failure reason.

**Comment by vranki** [comment]
Updated description - in future we hopefully have multiple ways of messaging user. Direct message via Matrix would be more realtime than e-mail.

**Comment by tswfi** [comment]
Not everyone has matrix account. Email is mandatory

**Comment by vranki** [comment]
Yes, naturally. But if user has matrix account configured, all the messages should be sent there. Any other messaging methods may become available in future.

**Comment by tswfi** [comment]
#233 does the email notification, closing this.
